squidpy/pl/_spatial.py

```diff
@@ -0,0 +1,415 @@
+                outline_params=outline_params,
+                cmap_params=cmap_params,
+                color_params=color_params,
+                size=_size,
+                color_vector=color_vector,
+                **kwargs,
+            )
+        elif _seg is not None and _cell_id is not None:
+            ax, cax = _plot_segment(
+                seg=_seg,
+                cell_id=_cell_id,
+                color_vector=color_vector,
+                color_source_vector=color_source_vector,
+                seg_contourpx=seg_contourpx,
+                seg_outline=seg_outline,
+                na_color=na_color,
+                ax=ax,
+                cmap_params=cmap_params,
+                color_params=color_params,
+                categorical=categorical,
+                **kwargs,
+            )
+
+        ax = _decorate_axs(
+            ax=ax,
+            cax=cax,
+            lib_count=_lib_count,
+            fig_params=fig_params,
+            adata=adata,
+            coords=_coords,
+            value_to_plot=value_to_plot,
+            color_source_vector=color_source_vector,
+            crops=_crops,
+            img=_img,
+            img_cmap=cmap_params.img_cmap,
+            img_alpha=color_params.img_alpha,
+            palette=palette,
+            legend_fontsize=legend_fontsize,
+            legend_fontweight=legend_fontweight,
+            legend_loc=legend_loc,
+            legend_fontoutline=legend_fontoutline,
+            na_color=na_color,
+            na_in_legend=legend_na,
+            scalebar_dx=scalebar_params.scalebar_dx,
+            scalebar_units=scalebar_params.scalebar_units,
+            scalebar_kwargs=scalebar_kwargs,
+        )
+
+    if fig_params.fig is not None and save is not None:
+        save_fig(fig_params.fig, path=save)
+
+
+@d.dedent
+@inject_docs(key=Key.obsp.spatial_conn())
+def spatial_scatter(
+    adata: AnnData,
+    shape: _AvailShapes | None = ScatterShape.CIRCLE,  # type: ignore[assignment]
+    img: _SeqArray | bool = True,
+    scalebar_kwargs: Mapping[str, Any] = MappingProxyType({}),
+    edges_kwargs: Mapping[str, Any] = MappingProxyType({}),
+    **kwargs: Any,
+) -> None:
+    """
+    Plot scatter plot of shapes in spatial coordinates.
+
+    %(plotting_shape_summary)s
+
+    %(plotting_general_summary)s
+
+    Parameters
+    ----------
+    %(adata)s
+    %(spatial_key)s
+    %(shape)s
+    %(color)s
+    %(groups)s
+    %(library_id)s
+    %(library_key)s
+    %(plotting_image)s
+    %(plotting_features)s
+    scalebar_kwargs
+        Keyword arguments for :func:`matplotlib_scalebar.ScaleBar`.
+    edges_kwargs
+        Keyword arguments for :func:`networkx.draw_networkx_edges`.
+    kwargs
+        Keyword arguments for :func:`matplotlib.pyplot.scatter` or :func:`matplotlib.pyplot.imshow`.
+
+    Returns
+    -------
+    %(plotting_returns)s
+    """
+    spatial_plot(
+        adata,
+        shape=shape,
+        img=img,
+        seg_key=None,
```

Comment:
Just a question: does this really need to set just 1 argument?

Reply:
`shape` and `img` have also different defaults.